demystifying_large_language_models/demystifying_large_language_models.md

```diff
@@ -0,0 +1,645 @@
+
+**Think about your own research, and write out how a large language model could improve or accelerate your work, and where it could introduce risk.**
+
+Clicking "submit" doesn't actually send this to us, and your answer isn't being evaluated in any way.
+
+[[___ ___ ___]]
+
+<details>
+
+<summary>Click here to see our answer</summary>
+
+<div class = "answer">
+
+We don't know what your research entails, or what your intentions are for the use of AI in your career.  Maybe you'll use it to help scaffold code or handle some of your unpublished writing tasks, such as drafting emails.  However, regardless of your use of large language models, we encourage you to:
+
+* Make no assumptions about the accuracy of any text generated by LLMs 
+* Disclose your use of LLMs
+* Investigate the policies of your institution, the journal you submit to, your granting agency, etc.
+* Consider not only the massive benefits of LLMs but also the very real risks that come along with using them
+
+</div>
+</details>
+
+
+## Additional Resources
+
+LLMs:
+
+* [ChatGPT (at the time of this module creation, built on GPT-3.5)](https://openai.com/chatgpt) 
+* [GPT-4](https://openai.com/gpt-4)
+* [Bing AI Chatbot (at the time of this module creation, built on GPT-4)](https://chat.bing.com)
+* [BERT](https://cloud.google.com/ai-platform/training/docs/algorithms/bert-start)
+
+Popular Press / Blogs:
+
+* [Large language models: their history, capabilities and limitations](https://snorkel.ai/large-language-models-llms/) is an overview by the company Snorkel that is appropriate for people who aren't experts in the field.
+* [An introduction to word embeddings for text analysis](https://www.shanelynn.ie/get-busy-with-word-embeddings-introduction/) builds intuition around words as vectors, with both common-sense explanations and great visualizations.
+* [The Illustrated Word2vec](http://jalammar.github.io/illustrated-word2vec/) is a superb page that explains word vectorization with both simplicity and depth.  This is a long piece but worth reading and watching (there's an embedded YouTube video with the author walking you through the article), as it builds your intuition about the intersection of words and their mathematical representation.
+
+Peer-reviewed literature:
+
+* [Ethical and social risks of harm from Language Models](https://arxiv.org/abs/2112.04359): a superb overview of the risk landscape in LLMs
+* [Attention is All You Need](https://arxiv.org/pdf/1706.03762.pdf): Google's work on Transformer models
+* [Distributed Representations of Words and Phrases and their Compositionality](https://proceedings.neurips.cc/paper_files/paper/2013/file/9aa42b31882ec039965f3c4923ce901b-Paper.pdf)
+
+History:
+
+* [The manuscript describing ELIZA](https://dl.acm.org/doi/10.1145/365153.365168)
+
+Drawing Things:
+
+* [academo's 3D Vector Plotter](https://academo.org/demos/3d-vector-plotter/)
+* [NN-SVG](http://alexlenail.me/NN-SVG/index.html)
+
```

Comment:
```suggestion
```
Not really related to the module topic, and if people really are interested in playing with the drawing stuff, you've already linked to these when you reference the sources within module content.